### PR TITLE
Feature/Tabs A11y [ED-1170, ED-1216]

### PR DESCRIPTION
**Pull request by kobizz**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/frontend/handlers/base-tabs.js [hunk 1]
```diff
@@ -13,6 +14,12 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 			toggleSelf: true,
 			hidePrevious: true,
 			autoExpand: true,
+			keyDirection: {
+				ArrowLeft: elementorFrontendConfig.is_rtl ? 1 : -1,
+				ArrowUp: -1,
+				ArrowRight: elementorFrontendConfig.is_rtl ? -1 : 1,
+				ArrowDown: 1,
+			},
```
Comment (matipojo) [inline assets/dev/js/frontend/handlers/base-tabs.js:22]: Maybe use `next` and `prev`?

File: includes/widgets/tabs.php [hunk 2]
```diff
@@ -392,31 +393,38 @@ protected function render() {
...
 				foreach ( $tabs as $index => $item ) :
 					$tab_count = $index + 1;
-
 					$tab_title_setting_key = $this->get_repeater_setting_key( 'tab_title', 'tabs', $index );
+					$tab_title = $a11y_improvements_experiment ? $item['tab_title'] : '<a href="">' . $item['tab_title'] . '</a>';
 
 					$this->add_render_attribute( $tab_title_setting_key, [
 						'id' => 'elementor-tab-title-' . $id_int . $tab_count,
 						'class' => [ 'elementor-tab-title', 'elementor-tab-desktop-title' ],
+						'aria-selected' => 1 == $tab_count ? 'true' : 'false',
```
Comment (matipojo) [inline includes/widgets/tabs.php:413]: Here and next comparisons .
```suggestion
						'aria-selected' => 1 === $tab_count ? 'true' : 'false',
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/handlers/base-tabs.js (hunk 1 above)
Comment (kobizz) [reply assets/dev/js/frontend/handlers/base-tabs.js:22]: These numbers are used for applying direction logic in the entire code

Comment (kobizz) [reply assets/dev/js/frontend/handlers/base-tabs.js:22]: Done

File: includes/widgets/tabs.php (hunk 2 above)
Comment (kobizz) [reply includes/widgets/tabs.php]: Done

**Review by matipojo — APPROVED** [review]

**Pull request merged**